components/pill-container/__examples__/bare.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import IconSettings from '~/components/icon-settings';
+import PillContainer from '~/components/pill-container';
+
+class Example extends React.Component {
+	static displayName = 'BarePillListboxExample';
+
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			options: [
+				{
+					bare: true,
```

Comment:
@interactivellama it can be both. The SLDS Listbox of Pills `.slds-pill_bare` modifier shows a class on the pill itself, but there's also a `.slds-pill_container_bare` class for the container. See below:
<img width="1088" alt="screen shot 2018-11-28 at 11 35 23 am" src="https://user-images.githubusercontent.com/991513/49173850-0a1da800-f302-11e8-8255-15a5744ea636.png">